HarpGuru/global.d.ts

```diff
@@ -15,15 +15,10 @@ import type { DisplayModes } from './types'
 
```

Comment:
I suppose the comment above this line isn't required any more, but I don't think it's unlikely that it will be required again, so to avoid needing to rewrite the thing I'll just leave it in place.